src/App.scss

```diff
@@ -158,7 +158,7 @@
 /* Query Suggestions */
 
 .uni-QuerySuggestions {
-  align-items: flex-start;
+  align-items: center;
```

Comment:
It would be better to keep `flex-start` and align manually for single liners to avoid this:

![Capture d’écran 2020-05-11 à 11 07 49](https://user-images.githubusercontent.com/5370675/81544344-afea9880-9377-11ea-990d-58ba1b1e9eda.png)